R/h_bracket.R

```diff
@@ -0,0 +1,21 @@
+#' @rdname h_bracket
+#' @export
+setMethod(
+    f = "h_bracket",
+    signature = "character",
+    definition = function(x) {
+       if(nchar(x) >= 1) paste0("{\n", paste0(x, collapse = "\n"), "\n}\n")
+    }
+)
+
+#' @rdname h_bracket
+#' @export
+setMethod(
+    f = "h_bracket",
+    signature = "list",
+    definition = function(x) {
+        model_text <- paste0(x, collapse = "\n")
+        paste0( model_text, "\n", "target+=sum(log_lik);\n")
```

Comment:
How come we are implicitly adding  `target+=sum(log_lik)` shouldn't this be explicitly part of the model code in the OS model ?

Reply:
Let me know If I understood your suggestion. The user will specify: 
`my_os_mod <- OsModel( stan = 
StanModule( model = "target+=sum(log_lik)" ))`. And then, when the Os and Long model will be merged, the final model section will use the prior list and combine it with the already existing model section code?

Reply:
Not quite, I would expect the  `StanModule( model = "target+=sum(log_lik)" )` to be apart of the default Stan code that is returned by the `LogLogisticOs` object